# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGL2Context.h
# Commit: 7e2a2b1b416f
# Full Hash: 7e2a2b1b416f1a64c6548aab29e92acc9554ae7d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix all webgl regression tests according to CI. r=handyman
#   
#   (This is a combination of 31 commits)
#   
#   * Fix Linux compilation.
# ==============================================================================

diff -r ccfa767dba64 -r 7e2a2b1b416f dom/canvas/WebGL2Context.h
--- a/dom/canvas/WebGL2Context.h	Wed Jan 08 22:19:16 2020 +0000
+++ b/dom/canvas/WebGL2Context.h	Wed Jan 08 22:19:23 2020 +0000
@@ -63,8 +63,6 @@
   // -------------------------------------------------------------------------
   // Texture objects - WebGL2ContextTextures.cpp
 
-  GLint GetFragDataLocation(const WebGLProgram& prog, const nsAString& name);
-
   // GL 3.0 & ES 3.0
   void VertexAttribI4i(GLuint index, GLint x, GLint y, GLint z, GLint w);
   void VertexAttribI4ui(GLuint index, GLuint x, GLuint y, GLuint z, GLuint w);